sp800-63b/appA_memorized.md

```diff
@@ -30,5 +30,12 @@ Users also express frustration when attempts to create complex passwords are rej
 
 Users’ password choices are very predictable, so attackers are likely to guess passwords that have been successful in the past. These include dictionary words and passwords from previous breaches, such as the “Password1!” example above. For this reason, it is recommended that passwords chosen by users be compared against a “black list” of unacceptable passwords. This list should include passwords from previous breach corpuses, dictionary words, and specific words (such as the name of the service itself) that users are likely to choose. Since user choice of passwords will also be governed by a minimum length requirement, this dictionary need only include entries meeting that requirement.
 
-### A.4 Summary
+### A.5 Randomly-chosen secrets
+
+Another factor that determines the strength of memorized secrets is the process by which they are generated. Secrets that are randomly chosen (in most cases by the verifier or CSP) and are uniformly distributed will be more difficult to guess or brute-force attack than user-chosen secrets meeting the same length and complexity requirements. Accordingly, at LoA 2, SP 800-63-2 permitted the use of randomly generated PINs with 6 or more digits while requiring user-chosen memorized secrets to be a minimum of 8 characters long.
```

Comment:
This is a really good comment, and I think it raises a question: Do we want to address the fact that a machine-chosen random password can be made easily human-memorable? I'm thinking of the world-list choice or even the poetry generator algorithm. We shouldn't normatively require anything like that but it might be worth it to mention the existence of this scheme.